### ISSUE TITLE
Could not start application xmerl: could not find application file: xmerl.app

### ISSUE THREAD

**Issue by hdcos** [kbrw/sweet_xml#22]
Hello when I install your package and run 

```
iex -S mix
```

I get the following error message from iex:

```
** (Mix) Could not start application xmerl: could not find application file: xmerl.app
```

Am I missing something?
I use elixir 1.2 and OTP 18

**Comment by hdcos** [comment]
Ok sorry I had to install erlang-xmerl on ubuntu

**Comment by seantanly** [comment]
@hdcos looks like the issue is resolved. Can you close this? :)

**Comment by kephas** [comment]
How is this closed? I don't see anywhere that the use needs to do anything but `mix deps.get`. Isn't this a bug?

**Comment by Shakadak** [comment]
Hi @kephas,

have you also encountered this problem ? xmerl is part of OTP (as can be seen here: https://erlang.org/doc/applications.html, and here: https://github.com/erlang/otp/tree/master/lib/xmerl). I suggest checking your erlang installation to see if anything is missing

**Comment by kephas** [comment]
As for @hdcos I had to install `erlang-xmerl` or I would also get `Could not start application xmerl: could not find application file: xmerl.app`. I'm using Debian Bullseye/Sid. The fact that `xmerl` is an OTP application doesn't seem to mean it's always present where Erlang is installed.

**Comment by Shakadak** [comment]
Alright, if I understand better your earlier comment, we need to improve the readme and the documentation to ask the user to ensure that his installation of erlang was done including xmerl

**Comment by kephas** [comment]
Exactly! Sorry if it wasn't clearer before. I hope the PR would do the trick. The needed package is called `erlang-xmerl` on several systems with `apt` and `yum`. I don't know about binary distributions on Windows and MacOS, I've found at least one issue on Github where someone on MacOS was missing `xmerl` in their setup.

**Comment by Shakadak** [comment]
Thanks, I'll keep the issue open for a bit as a reminder for myself to see how I deal with this and the doc for https://hexdocs.pm/
